### PR TITLE
docs: rearrange root-level sidebar

### PR DESCRIPTION
**Pull request by galenmarchetti**
## Description:
Rearranging root-level sidebar to make reference material locations more obvious at a first glance, as a first step to more in detail reference doc revamp

## Is this change user facing?
YES

### CODE REVIEW

File: docs/docs/runtime-api-reference.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
-title: SDK
-sidebar_label: SDK
-slug: /SDK
+title: Runtime API Reference
```
Comment (leeederek) [inline docs/docs/runtime-api-reference.md]: @galenmarchetti - NACK from me re: this name change. We should keep it as SDK.

Context: We previously had this section titled as `Client Libraries` and more than a few people asked me where our SDK was (Chainlink asked, and also a user asked too when we were doing a user feedback session). I changed the wording to SDK with Kevin's +1 [here](https://kurtosistech.slack.com/archives/C04MBCY49HR/p1686151655619479). Another data point we got recently was that our users search for the "Kurtosis SDK" on Google - https://github.com/kurtosis-tech/kurtosis/issues/1238

File: docs/sidebars.js [hunk 2]
```diff
@@ -14,57 +14,63 @@
...
         {
             "type": "link",
             "label": "Kurtosis for Web3",
             "href": "https://web3.kurtosis.com",
         },
-        'quickstart',
-        'code-examples',
         {
             type: 'category',
-            label: 'Guides',
```
Comment (leeederek) [inline docs/sidebars.js:27]: Would prefer the `Guides` section be near the top of the sidebar, proposing that it be right below the `Quickstart`

**Review by leeederek — CHANGES_REQUESTED** [review]
NACK from me on both the ordering and the name change from SDK --> runtime api reference

File: docs/docs/runtime-api-reference.md (hunk 1 above)
Comment (h4ck3rk3y) [reply docs/docs/runtime-api-reference.md]: Would prefer keeping the title as SDK as well

Comment (mieubrisse) [reply docs/docs/runtime-api-reference.md:2]: +1

Comment (galenmarchetti) [reply docs/docs/runtime-api-reference.md]: @h4ck3rk3y @leeederek  Roger - I'm looking for a specifier that identifies why "SDK" is different than _starlark_. With just SDK we don't give any information on a first glance about why there's a difference between starlark api and SDK. Thoughts on "runtime SDK" or <some other word> SDK that clarifies why we have an SDK _and_ a starlark API?

File: docs/sidebars.js (hunk 2 above)
Comment (galenmarchetti) [reply docs/sidebars.js:27]: No issue with that will change

File: docs/docs/runtime-api-reference.md (hunk 1 above)
Comment (h4ck3rk3y) [reply docs/docs/runtime-api-reference.md]: Interesting, I see what you are getting at. I can get on board with `runtime SDK` - So Starlark is pure APIC no engine, it all runs inside an APIC, while the current SDK is both.

Starlark - InsdieEnclaveSDK
SDK - Everything but EnclaveSDK

**Review by leeederek — APPROVED** [review]

**Pull request merged**